### ISSUE TITLE
Fix `alpha-value-notation` false positives for `color()`

### ISSUE THREAD

**Issue by mrcgrtz** [stylelint/stylelint#6884]
### What steps are needed to reproduce the bug?

```css
body {
	color: color(display-p3 1 1 0);
}
```

### What Stylelint configuration is needed to reproduce the bug?

```json
{
	"rules": {
		"alpha-value-notation": "percentage"
	}
}
```

### How did you run Stylelint?

[Stylelint demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIBG0CeACYAdAdlgLpADYQBO86xZAFFAJYDOADkQIaoC0TAzOgIz90ABgCUAblwEq5ShBKlajFuy68BAoegD0-IUICk43AF9cIADQgAZnSIwAcqwC2cRDAAezljAB0YBgwW4BA4NgDmCCDYOOjoWCAe+DA4UAzxFADauLGx8Qz4qHZEdDj4HJChdGEc+awprKRQ8dnoALomQRXhAGJkTqz4kQBWDCFBsEyBiNG5IPmFMMWl6XEgbEn58eYteQVFJWVdVTX4dVANTQir6zCbIB3GQA)

### Which version of Stylelint or dependencies are you using?

15.6.2

### What did you expect to happen?

No problems to be reported.

### What actually happened?

I got an error:

`Expected "0" to be "0%" (alpha-value-notation)`

### Does the bug relate to non-standard syntax?

No, it's modern syntax from [CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/#color-function).

**Comment by romainmenke** [comment]
Thank you for reporting this @mrcgrtz

Can confirm the issue and I've prepared a fix.
